MultiPage-Clone/main.js

```diff
@@ -1,4 +1,7 @@
+const burgerr = document.querySelector('.navimage');
+const nav = document.querySelector('.nav-links');
 
-const OVERFLOW = $('#toggle1');
-
-OVERFLOW.click(e => e.target.checked ? $("body").css("overflow-y", "hidden") : $("body").css("overflow-y", "auto"));
\ No newline at end of file
+burgerr.addEventListener('click', () => {
```

Comment:
You want that the link changes the way it looks when clicked, right?
Have a look at CSS :active selector https://www.w3schools.com/cssref/sel_active.php. 
Probably you don't need javascript for that at all.